packages/node_modules/@node-red/editor-client/src/js/nodes.js

```diff
@@ -568,8 +568,24 @@ RED.nodes = (function() {
         return node;
     }
 
+    function collectNodesInSubflow(n) {
+        var nodes = RED.nodes.filterNodes({z:n.id});
+        return createExportableNodeSet(nodes);
+    }
+    
     function convertSubflow(n) {
         var node = {};
+        var id = n.id;
+        if (n.flow) {
+            var flow = collectNodesInSubflow(n);
+            node.flow = flow;
+            node.exportName = n.exportName;
```

Comment:
Where has `exportName` come from? What role does it play?

Reply:
exportName represents corresponds to name in `Export as` input field (`Module` field in new revision).